.circleci/config.yml

```diff
@@ -10,45 +10,36 @@ dependencies:
   pre:
     - yarn global add npm
 jobs:
-  validate:
-    <<: *defaults
-    steps:
-      - run:
-          name: "Checking Versions"
-          command: |
-            node --version
-            npm --version
-            yarn --version
```

Comment:
why?

Reply:
yarn version won't print the correct version anymore, This script isn't providing value really.

Reply:
`yarn -v` it is

Reply:
Maybe comment it instead? To reenable once docker image gets updated

Reply:
Meh, it's really not useful the info is in the logs anyway.